gateway-js/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 > The changes noted within this `vNEXT` section have not been released yet.  New PRs and commits which introduce changes should include an entry in this `vNEXT` section as part of their development.  When a release is being prepared, a new header will be (manually) created below and the appropriate changes within that release will be moved into the new section.
 
--  _Nothing yet! Stay tuned!_
+- OpenTelemetry support PR TBA
```

Comment:
Let's update this with the PR # and link. I think the description is fine but if you feel there's anything worth adding here that's great too.